pkg/plugin/v2/api.go

```diff
@@ -114,6 +114,7 @@ func (p *createAPISubcommand) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&p.resource.Kind, "kind", "", "resource Kind")
 	fs.StringVar(&p.resource.Group, "group", "", "resource Group")
 	fs.StringVar(&p.resource.Version, "version", "", "resource Version")
+	fs.StringVar(&p.resource.Plural, "plural", "", "resource kind irregular plural form")
```

Comment:
as we spoke today shows fine we move forward with the new flag but for v3+ only

Reply:
I already removed this